tests/components/reolink/conftest.py

```diff
@@ -0,0 +1,62 @@
+"""Setup the Reolink tests."""
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+TEST_HOST = "1.2.3.4"
+TEST_HOST2 = "4.5.6.7"
+TEST_USERNAME = "admin"
+TEST_USERNAME2 = "username"
+TEST_PASSWORD = "password"
+TEST_PASSWORD2 = "new_password"
+TEST_MAC = "ab:cd:ef:gh:ij:kl"
+TEST_PORT = 1234
+TEST_NVR_NAME = "test_reolink_name"
+TEST_USE_HTTPS = True
+
+
+@pytest.fixture(name="reolink_setup_entry")
+def reolink_setup_entry_fixture() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.reolink.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture(name="reolink_connect")
+def reolink_connect_fixture(mock_get_source_ip):
```

Comment:
You can also drop the "name" argument if you drop "_fixture" from the function name.
It used to make sense when they were inside the test file, but it's not needed inside the conftest file anymore.
```suggestion
@pytest.fixture
def reolink_connect(mock_get_source_ip: None) -> Generator[MagicMock, None, None]:
```